roles/libvirt-setup/defaults/main.yml

```diff
@@ -30,20 +33,29 @@ libvirt_default_pool_dir: "{{libvirt_images_dir}}/pool-default"
 
 external_network_cidr_v4: 192.168.125.0/24
 external_network_bridge_ip: "{{ external_network_cidr_v4|ansible.utils.nthhost(1)|default('', true) }}"
```

Comment:
I guess renaming it to "_ip_v4" might result in many edits elsewhere. Not a strong preference but maybe it makes sense to keep it consistent alike with the other variables.

Reply:
Renamed to `external_network_bridge_ip_v4`